code-challenges.test.js

```diff
@@ -14,47 +22,148 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
+describe("fibArray" , () => {
+  it("takes in a number greater than 2 and returns an array with the same # of characters in the Fibonacci sequence.", () => {
+    const fibLength1 = 6
```

Comment:
Great job hoisting your variables inside of your tests, this is a really good practice!